testing/tests/DevExpress.ui.widgets/tabs.tests.js

```diff
@@ -656,6 +658,30 @@ QUnit.module('Horizontal scrolling', () => {
         assert.ok(itemOffset <= rightBoundary, `item offset ${itemOffset} is lower than right boundary ${rightBoundary}`);
         assert.ok(itemOffset > contentLeft, `item offset ${itemOffset} is greater than left boundary ${contentLeft}`);
     });
+
+    QUnit.test('tabs should scroll to the disabled item when it have focus', function(assert) {
+        const items = [{ text: 'item 1' }, { text: 'item 2' }, { text: 'item 3', disabled: true }];
+        const $element = $('#scrollableTabs').dxTabs({
+            items: items,
+            scrollingEnabled: true,
+            visible: true,
+            width: 200,
+            showNavButtons: false,
+            focusStateEnabled: true,
+        });
+        const $item = $element.find('.' + DISABLED_STATE_CLASS).eq(0);
```

Comment:
we usually use string literals as a more modern and short approach
```suggestion
        const $item = $element.find(`.${DISABLED_STATE_CLASS}`).eq(0);
```

Reply:
Yes, of course, sorry for the copy-paste. It was fixed in this commit [87ee87bc1c61b02d65c6472f29438a5a1611e46f].